apps/researcher/src/app/[locale]/objects/[id]/metadata.tsx

```diff
@@ -0,0 +1,93 @@
+import {HandThumbUpIcon} from '@heroicons/react/24/outline';
+import {useTranslations} from 'next-intl';
+import {ReactNode} from 'react';
+import {create} from 'zustand';
+
+export const useCurrentOwner = create(() => ({
+  name: '',
+  location: '',
+  dataKey: '',
+}));
+
+interface Props {
+  dataKey: string;
```

Comment:
What is a 'data key'? Would it be possible to use a more specific or functional name?